cmd/promcord/service.go

```diff
@@ -74,6 +95,12 @@ func main() {
 	if err := view.Register(MsgCountView); err != nil {
 		log.From(ctx).Fatal("registering views", zap.Error(err))
 	}
+	if err := view.Register(MsgLengthView); err != nil {
+		log.From(ctx).Fatal("registering views", zap.Error(err))
```

Comment:
Here one could add a tag for logging more detail about the failing action
like: `zap.String("view", MsgLengthView.Name)`
Further refactoring this it would be possible to pull the Register call inside a for loop and list all Views to register inside an array.